src/common/sorted-table/index.js

```diff
@@ -252,26 +253,28 @@ export default class SortedTable extends Component {
             ))}
           </tbody>
         </table>
-        {filterContainer
-          ? (
-          <Portal container={() => filterContainer()}> // Rebuild container function to refresh Portal component.
-            {filterInstance}
-          </Portal>
-          ) : (
-          <Row>
+        <Container>
+          <SingleLineRow>
```

Comment:
Maybe this row should not exists if both `paginationContainer` and `filterContainer` is provided.